formatTest/unit_tests/expected_output/modules.re

```diff
@@ -515,24 +516,42 @@ module M = {
 
 module N = {
   open M;
-  let z = M.(34);
+  let z = {
+    M.(34);
+  };
   let z = {
     open M;
     34;
     35;
   };
+  let z = {
+    M.(34, 35);
```

Comment:
Same for every local open sugar, is this part of this PR? I'm not sure that we want braces if there's an open sugar transformation happening in printing